src/mbgl/text/collision_index.hpp

```diff
@@ -20,7 +20,7 @@ class CollisionIndex {
 public:
     using CollisionGrid = GridIndex<IndexedSubfeature>;
 
-    explicit CollisionIndex(const TransformState&);
+    explicit CollisionIndex(const TransformState&, const MapMode&);
```

Comment:
nit: MapMode is a enum, no need for const reference

Reply:
fixed in https://github.com/mapbox/mapbox-gl-native/pull/15880/commits/d733f53e8dfe47f1ef7f3667ab7de188bbc3cc91